wled00/xml.cpp

```diff
@@ -271,6 +281,14 @@ void getSettingsJS(byte subPage, char* dest)
     DynamicJsonDocument doc(JSON_BUFFER_SIZE/2);
     JsonObject mods = doc.createNestedObject(F("um"));
     usermods.addToConfig(mods);
+
+    // By using the pin manager, all pins that are currently owned
+    // are exposed via a common API.  Zero == unallocated, value
+    // corresponds to the current owner's value (see enum in pinmanager.h)
+    oappend(SET_F("d.pinOwners=["));
```

Comment:
For settings_led.htm, I'll adjust my output variable to be `d.um_p` instead, and only list allocated pins as their value.

For settings_um.htm, I first have to say I'm impressed with the dynamic user mod configuration support.  Wow!

<HR/><details><summary>At the same time, I'd need help... (click to expand)</summary><P/>

What is going on in this page is a bit hard for me to parse.

I believe the following occurs in the script:

1. `pins` is the list of pins that are allocated
2. `pinO` is a list of strings that somehow (`????`) correspond to which part of `cfg.json` listed it(?)
3. function `getPins()` pushes new pin numbers into `pins` and some sort of `owner` designator into `pinO`.  I know `owner` corresponds to my `????`, but I don't grok where that value is obtained.
4. function `check()` allows pins that are not already in `pinO` array.  It will also allow pins that are in that array, but only when (`????`) matches the original owner (looks up appropriate index in `pins`, compares (`????`) against corresponding index in `pinO`).
</details><HR/>

Because I'm not clear on how the dynamic ID generation works, nor how the `pinO` values are generated, I'm also not entirely clear how to modify it using the information from `pinManager` to better avoid accidental attempts to allocate pins that are already in use.  Ideas?